scripts/error-extractor/__tests__/evalToString-test.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+var evalToString = require('../evalToString');
+var babylon = require('babylon');
+
+var parse = (source) => babylon.parse(
+  `invariant(${source});`
+).program.body[0].expression.arguments[0]; // quick way to get an exp node
```

Comment:
(I don't think you need the invariant here; `(${source});` would probably be enough.)